src/frontend/src/components/card/SubEmotionTab.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <v-container pa-0 ma-0>
+    <v-chip-group
+      active-class="black--text text--accent-4"
+      column
+      multiple
+      align="right"
+      justify="end"
+    >
+      <v-chip v-for="subEmotion in subEmotions" :key="subEmotion.id">
+        # {{ subEmotion.name }}
+      </v-chip>
+    </v-chip-group>
+  </v-container>
+</template>
```

Comment:
피드에서 터치로 쓱쓱 화면을 내릴때 
소분류 감정 태그때문에 화면이 안내려가질때가 있더라구요
터치 시작점이 소분류 감정 태그면 태그 터치로 인식해서 스크롤이 안되는 것 같아요
v-chip-group의 특성인것 같아서 v-chip-group을 지우고 이것저것 만졌더니 잘 되네요!!

```suggestion
<template>
  <v-container>
    <v-row>
      <v-chip
        v-for="subEmotion in subEmotions"
        :key="subEmotion.id"
        class="mr-2"
      >
        # {{ subEmotion.name }}
      </v-chip>
    </v-row>
  </v-container>
</template>

```

Reply:
테스트 감사합니다 💯 
안그래도 로직 변경으로 v-chip-group을 쓸 필요가 없어졌는데
이슈까지 있으니...아예 제거하는 방향으로 가볼게요 감사합니다 👍